src/omnisharp/OmnisharpPackageCreator.ts

```diff
@@ -5,48 +5,56 @@
 
 import { Package } from "../packageManager/Package";
 
-export function GetPackagesFromVersion(version: string, runTimeDependencies: Package[], serverUrl: string, installPath: string): Package[] {
+export const modernNetVersion = "6.0";
```

Comment:
Filed https://github.com/OmniSharp/omnisharp-roslyn/issues/2294 to track removing the need for this constant.